code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -99,9 +99,12 @@
 /obj/structure/constructshell/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/device/soulstone))
 		var/obj/item/device/soulstone/SS = O
+		if(!iscultist(user) && !iswizard(user))
```

Comment:
oh right I'm an idiot and somehow forgot the whole reason for putting it under line 105

if(!iscultist(user) && !iswizard(user) && !SS.usability)